lib/publiccloud/ssh_interactive.pm

```diff
@@ -1,67 +1,98 @@
 # SUSE's openQA tests
 #
-# Copyright 2019 SUSE LLC
+# Copyright 2022 SUSE LLC
 # SPDX-License-Identifier: FSFAP
 
 # Summary: Class with helpers related to SSH Interactive mode
 #
-# Maintainer: Pavel Dostal <pdostal@suse.cz>
+# Maintainer: qa-c@suse.de
 
 package publiccloud::ssh_interactive;
+use testapi qw(is_serial_terminal);
 use base Exporter;
 use testapi;
 use Utils::Backends qw(set_sshserial_dev unset_sshserial_dev);
+use utils qw(type_string_very_slow);
 use strict;
 use warnings;
+use distribution;
 
 our @EXPORT = qw(ssh_interactive_tunnel ssh_interactive_leave select_host_console);
 
+sub establish_tunnel_console {
+    # Helper call to activate a console and establish the ssh connection therein
+    my $console = $_[0];
```

Comment:
```suggestion
    my ($console) = @_;
```
I am really bad with all this special perl vars , but isn't that equal to what I wrote ? If yes I would prefer this more common version which is used commonly in this repo